scripts/seed.py

```diff
@@ -175,11 +175,15 @@ def create_challenge_phases(challenge, number_of_phases=1):
     challenge_phases = []
```

Comment:
Can you please open up a new PR with the changes in this file?

Reply:
Done